### PR TITLE
Documented tags more

### PR DESCRIPTION
**Pull request by WebFreak001**

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -70,34 +70,97 @@ A tag consists of the following parts, which, if present, must occur in the list
...
+    </foo>
+    ```
     - A single `>` will instruct the generator not to emit additional white space around the generated HTML element.
+    ```html
+    div
+    foo<
+      a bar
+    // Generates
+    <div></div>
+    <foo><a>bar</a></foo>
```
Comment (s-ludwig) [inline SPEC.md]: Correction:

    <div></div><foo>
        <a>test</a>
    </foo>

File: SPEC.md [hunk 2]
```diff
@@ -70,34 +70,97 @@ A tag consists of the following parts, which, if present, must occur in the list
...
+        return text == "Hello, World!" ? "Hallo, Welt!" : text;
+      }
+    }
+    auto dst = appender!string;
+    dst.compileHTMLDietFile!("diet.dt", Context);
+    ```
+
+    or when using inside vibe.d you use it with a `translationContext`.
+
+Alternatively to a tag you may also place a `| text` node which will insert the raw text (`text` in this case) into the HTML document. You can use this to combine tags into text or foreach loops adding text. Adding a second space will start inserting actual spaces into the inserted text as only everything after the `| ` is consumed, see [Text nodes](text-nodes).
```
Comment (s-ludwig) [inline SPEC.md]: There appears to be a verb missing at the beginning of the sentence here.

File: SPEC.md [hunk 3]
```diff
@@ -70,34 +70,97 @@ A tag consists of the following parts, which, if present, must occur in the list
...
 
     - A single `<` will instruct the generator not to emit additional white space within the generated HTML element.
+    ```html
+    div
+    foo>
+      a bar
+    // Generates
+    <div></div><foo>
+      <a>bar</a>
+    </foo>
```
Comment (WebFreak001) [inline SPEC.md]: Uh I think I just took those from unittests, but I can change it if I accidentally took the wrong ones

**Comment by s-ludwig** [comment]
Thanks!

**Pull request merged**